http/src/main/resources/reference.conf

```diff
@@ -552,8 +552,11 @@ pubsub {
   }
 
   non-leo-message-subscriber {
-    subscription-name = "nonLeoMessageSubscription"
+    # subscription-name = "nonLeoMessageSubscription"
     dead-letter-topic = "leoDeadLetterTopic"
+    # Topic that we publish cryptomining users to.
+    # Bard and Sam will subscribe to this topic and act upon it accordingly
+    terra-cryptomining-topic = "terra-cryptomining"
```

Comment:
Was wondering if we'd need a topic-per-fiab like we do for other topics, but it probably doesn't matter here because we won't have automation tests that do cryptomining.

Reply:
yea...there's no automation test for cryptomining yet...I was thinking about it, but figured we can add it if we run into a case where without automation test it becomes a problem since it'll be a bit of work to make a cryptomining automation test